content/blog/announcing-enum-support/index.md

```diff
@@ -0,0 +1,277 @@
+
+### "Strict" enums
+
+A property is a "strict" enum when the input value **must** be one of the enumerated values. In this case, the property type is specified as the enum type. Currently, only the Azure NextGen provider uses "strict" enums as the OpenAPI specification defines the properties as such.
+
+{{< chooser >}}
+{{< choosable language typescript >}}
+
+```typescript
+// Add lang-specific representation of a strict enum property
+```
+
+{{< /choosable >}}
+{{< choosable language python >}}
+
+```python
+# Add lang-specific representation of a strict enum property
+```
+
+{{< /choosable >}}
+{{< choosable language csharp >}}
+
+```csharp
+// Add lang-specific representation of a strict enum property
+```
+
+{{< /choosable >}}
+{{< choosable language go >}}
+
+```go
+// Add lang-specific representation of a strict enum property
+```
+
+{{< /choosable >}}
+{{< /chooser >}}
+
+### "Relaxed" enums
+
+When a property is a "relaxed" enum, the property type is specified as the `Union` of the enum type and the underlying primitive type. This means that you have the convenience of using the enum constants (i.e. `s3.CannedAcl.Private`), but you may also pass in the raw string (i.e. `"private"`).
+
+In the AWS provider (and other Terraform-based providers) we have opted for **only** using "relaxed" enums. The reasoning here is twofold. Allowing the primitive type maintains backwards compatibility and also allows users to use values which may not yet be represented in the Pulumi schema (e.g. a new Managed Policy ARN, EC2 Instance type, etc.).
+
+{{< chooser >}}
+{{< choosable language typescript >}}
+
+```typescript
+// Add lang-specific representation of a relaxed enum property
+```
+
+{{< /choosable >}}
+{{< choosable language python >}}
+
+```python
+# Add lang-specific representation of a relaxed enum property
+```
+
+{{< /choosable >}}
+{{< choosable language csharp >}}
+
+```csharp
+// Add lang-specific representation of a relaxed enum property
+```
+
+{{< /choosable >}}
+{{< choosable language go >}}
+
+```go
+// Add lang-specific representation of a relaxed enum property
+```
+
+{{< /choosable >}}
+{{< /chooser >}}
+
+## Try them out!
+
+You can find enum types integrated into v3.19.0 of the AWS provider, and v0.3.0 of the Azure NextGen provider.
+
+In the Azure NextGen provider, all properties labelled as enums in the OpenAPI spec are represented as such. On the other hand, the enums for the AWS provider are manually identified and maintained as part of the [provider schema](https://github.com/pulumi/pulumi-aws/blob/master/provider/resources.go#L2392-L3375).
+
+We've already added many that you might find useful, such as Lambda Runtimes, EC2 Instance Types and IAM Managed Policies, and will continue to add more across our provider SDKs in the coming months.
```

Comment:
I would restructure the above paragraphs to have one about AWS and one Azure. For Azure, we may throw a fancy stat like "we have 1000 enums and 5000 enum values for you" (I made up the numbers, should be an easy query at the schema level).